test/doe_prototype/test_performance.rb

```diff
@@ -5,7 +5,7 @@ class TestDOEPrototypesPerformance < CreateDOEPrototypeBuildingTest
   @building_types = [
     'SmallOffice',
     'MediumOffice',
-    'LargeOffice',
+#    'LargeOffice',
```

Comment:
Temporary (or definitely?) deactivating performance testing for large office until [this issue](https://github.com/NREL/EnergyPlus/issues/10140) is addressed.